src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -162,6 +162,13 @@ public PlatformDetectorResult Detect(RepositoryContext context)
 
             var manifestFileProperties = new Dictionary<string, string>();
 
+            string runBuildCommand = Environment.GetEnvironmentVariable("RUN_BUILD_COMMAND");
```

Comment:
nit: I'd suggest pulling the `"RUN_BUILD_COMMAND"` string out into its own private `const` defined at the top of this file.